dataactcore/scripts/pullFPDSData.py

```diff
@@ -943,6 +944,9 @@ def calculate_remaining_fields(obj, sess, sub_tier_list, county_by_name, county_
         except KeyError:
             unique_string += "-none-"
 
+    # calculate business categories
+    obj['business_categories'] = get_business_categories(row=obj, data_type='fpds')
```

Comment:
Since `unique_string` is getting built above this it would make sense for it also to get set above this (either that or both things to happen below this)